doc/pod_chaos.md

```diff
@@ -0,0 +1,70 @@
+# Pod Chaos Document
+
+This document helps you to build pod chaos experiments. 
+
+Pod chaos allows you to simulate pod faults, specifically `pod failure` and `pod kill`.
```

Comment:
can you supplement some details about which case does `Pod failure` suit for use and which case does `Pod Kill` suit for?